tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/jdbc/JdbcTableManager.java

```diff
@@ -73,18 +79,77 @@ public TableInstance createImmutable(JdbcTableDefinition tableDefinition, TableH
     {
         TableName tableName = createImmutableTableName(tableHandle);
         LOGGER.debug("creating immutable table {}", tableName);
-        dropTableIgnoreError(tableName); // ultimately we do not want to drop table here - if data did not change
-        createTable(tableDefinition, tableName);
-        JdbcTableDataSource dataSource = tableDefinition.getDataSource();
-        insertData(tableName, dataSource);
+        // TODO: drop and recreate table if the underlying data has changed
```

Comment:
But for the TPCH tables, there is no version number? I'm going to leave this be for now, I think it means adding some additional infrastructure.